package.json

```diff
@@ -57,17 +59,18 @@
     "underscore": "1.8.3"
   },
   "devDependencies": {
+    "deep-equal": "~1.0.1",
     "grunt": "~1.0.1",
     "grunt-fh-build": "~2.0.0",
     "istanbul": "0.4.3",
+    "license-reporter": "github:bucharest-gold/license-reporter#a64f5f7488e447c17327eef256dc40979eb0d3c4",
```

Comment:
This is the latest commit on master of the license-reporter repo. It's not yet published to npmjs, and specifying master is not stable (from experience -- our Jenkins builds already run this tool, and previously depended on master, and broke frequently as a result).

Reply:
Yes, +1 to publish the component to npmjs, once it reaches 1.0 milestone :-)

Reply:
That's the plan! Jay is looking after this as we might want to name it appropriately to show it's Red Hat supported (legal get involved) and / or namespace it.